src/editors/Flow/view/Components/interface/canvas.js

```diff
@@ -704,10 +692,11 @@ class Canvas {
       .duration(750)
       .call(
         this.zoomBehavior.transform,
-        d3.zoomIdentity
+        (d3.zoomIdentity
           .translate(width / 2, height / 2)
-          .translate(-SCALE * xCoordinate, -SCALE * yCoordinate)
+          .translate(- SCALE * xCoordinate, - SCALE * yCoordinate)
```

Comment:
I'm curious, doesn't this space change the calculation?

Reply:
I don't think so.